src/main/java/org/alfresco/repo/management/subsystems/CryptodocSwitchableApplicationContextFactory.java

```diff
@@ -128,23 +65,80 @@ public void setProperty(String name, String value)
         {
             if (!isUpdateable(name))

```

Comment:
Why do we need this test? I cannot see that `isUpdateable` is overwritten anywhere, so it should return the default `true`.
And even if it is overwritten, I think the message on the Exception is wrong.

Reply:
True, I removed the test.